code-challenges.test.js

```diff
@@ -26,26 +30,194 @@ const colors2 = [
+    
+    // expect statement will contain the function call to call the given functions,
+    // join the two arays to return only one array
+    // place test variable outputs inside the .toEqual() matcher parameter to compare the given outputs
+      expect(values(dataArray1, dataArray2)).toEqual(["array", "object", "number", "string", "Boolean", "null", "undefined"])
+  })
+})
+
+//FAIL!: output - values is not defined
+// FAIL  ./code-challenges.test.js
+// values
+//   ✕ takes in two arrays as arguments and returns one array with no duplicate values. 
+
+// ● values › takes in two arrays as arguments and returns one array with no duplicate values. 
+
+//   ReferenceError: values is not defined
 
-const dataArray1 = ["array", "object", "number", "string", "Boolean"]
-const dataArray2 = ["string", "null", "Boolean", "string", "undefined"]
-// Expected output: ["array", "object", "number", "string", "Boolean", "null", "undefined"]
 
 // b) Create the function that makes the test pass.
+
+// Process: GREEN!
+
+// PSEUDOCODE:
+// Create a function that takes in two arrays as arguments and returns one array with no duplicate values using the spread operator to pass in a dynamic number of arguments.
+
+// Input: two arrays as arguments ("takes in.")
+// Output: returns one array with no duplicate values.
+
+// Given arrays:
+// Input (array1): ["array", "object", "number", "string", "Boolean"]
+// Input (array2): ["string", "null", "Boolean", "string", "undefined"]
+// Output: ["array", "object", "number", "string", "Boolean", "null", "undefined"]
+
+// Create a function named "values" that takes in both given arrays as arguments using the spread operator naming it "...arrays".
+//Combine the given arrays by naming the function 'combinedArrays', assigning an empty array and then join both arrays using the .concat() method and passing the arrays as an argument using the spread operator. 
+// Create a 'newArray' function for your newly combined arrays and assign it by using the 'Array.from' constructor method which will take any object in the arrays, iterate over each element in the arrays and return values in the arrays. Then create a 'new Set' constructuctor method to create one set of both arrays to iterate over which will remove the duplicate elements of the combined arrays. 
+// Call on the last 'newArray' function that holds all of the completed logic of the functions and given arrays. 
+//Returns one array with no duplicate values.
+
+const values = (...arrays) => {
+  let combinedArrays = [].concat(...arrays)
+  let newArray = Array.from(new Set(combinedArrays))
+  return newArray
+}
+// PASSED!
+// PASS  ./code-challenges.test.js
+// values
+//   ✓ takes in two arrays as arguments and returns one array with no duplicate values.  (1 ms)
+
+// Test Suites: 1 passed, 1 total
+// Tests:       3 passed, 3 total
+// Snapshots:   0 total
+// Time:        0.131 s, estimated 1 s
```

Comment:
Good job researching different tools available to you and implementing them here! Great use of `...`, `Array.from()`, and `new Set()`!
Also, same as the last comment, don't need to create the newArray variable, you can just return the data itself!